kratos/geometries/line_2d_2.h

```diff
@@ -583,7 +583,8 @@ class Line2D2 : public Geometry<TPointType>
     */
     JacobiansType& InverseOfJacobian( JacobiansType& rResult, IntegrationMethod ThisMethod ) const override
     {
-        KRATOS_ERROR << "Jacobian is not square" << std::endl;
+        rResult[0] = ZeroMatrix( 1, 1 );
```

Comment:
I would say that this inverse is not defined since the jacobian is a rectangular matrix. I see that the Triangle3D2 is also returning an error here

Reply:
I saw the code in the line3d and I copied in the 2D. In any case it is technially posible to compute the generalized inverse, we have it implemented on the math_utils in fact